### ISSUE TITLE
Events "buddylist_overlay" and "chatproxy-presence" never received

### ISSUE THREAD

**Issue by gave92** [carpedm20/fbchat#460]
## Description of the problem
When listening, events "buddylist_overlay" and "chatproxy-presence" are never received, so the `getUserActiveStatus()` method always returns None. Is it just me?

## Code to reproduce
```py
from fbchat import Client
client = Client("email", "pw")
client.listen(markAlive=True)
```

## Environment information
- `fbchat` 1.7.3

On a brighter note I found that the following request:
```
POST https://m.facebook.com/buddylist_update.php HTTP/1.1
Host: m.facebook.com
Cookie: ...

data_fetch=true&send_full_data=true&THEUSUALPAYLOADSTUFF
```
returns the list of ids of the currently active friends if the following form (status=0 is active/idle):
```
{
  "__ar": 1,
  "payload": {
    "buddylist": [
      {
        "id": 100000441126604,
        "status": 0
      },
      {
        "id": 1871422185,
        "status": 0
      },
      ...
```

**Comment by madsmtm** [comment]
Hmm, concerning, I'll have a closer look when I get the time. Possibly related to #430?

**Comment by gave92** [comment]
Well monitoring the websocket I'm seeing this messages, so it's possible:
`.../orca_presence{"list_type":"full","list":[{"u":XXXXXXXXX,"p":0,"l":1566490349},{"u":XXXXXXXXX,"p":2,"c":9969664},{"u":XXXXXXXXX,"p":0,"l":1566493521},{"u":XXXXXXXXX,"p":0,"l":1566494154,"c":9961488}...`

**Comment by vincent-lg** [comment]
The given request is useful.  Trouble is, if you want to be up-to-date on who's online and who's not you have to send this request a lot.  It would be great if the buddylist_overlay event (or whatever is being used) would work in that situation.

**Comment by madsmtm** [comment]
Should be fixed in #494, release `v1.9.0`
